docs/general/serializers.md

```diff
@@ -136,7 +136,97 @@ PR please :)
 
 #### #scope
 
-PR please :)
+Allows you to include in the serializer access to an external method.
+
+It's intended to provide an authorization context to the serializer.
+
+`scope` refers to the name of a method in the serialization context, that
+may be made available to the serializer instance. It is the value of the `scope:` option
+to the new serializer.
+
+When the `scope_name:` option is passed to the new serializer, and a method by that
+name does not already exist, the serializer defines a method with that name that calls
+the `scope` method.
+
+That's a lot of words, so here's some examples:
+
+First, let's assume the serializer is instantiated in the controller, since that's the usual scenario.
+We'll refer to the serialization context as `controller`.
+
+| options | `Serializer#scope` | method definition |
+|-------- | ------------------|--------------------|
+| `scope: :current_user, scope_name: :current_user` | `:current_user` | `Serializer#current_user` calls `controller.current_user`
+| `scope: :current_admin, scope_name: :current_admin` | `:current_admin` | `Serializer#current_admin` calls `controller.current_admin`
+
+Since the method is dynamically defined, there is no concern with it being called when not used.
+This was a problem in
+[`0.9`](https://github.com/rails-api/active_model_serializers/tree/0-9-stable#customizing-scope)
+when the [`current_user` method was being called on every request](https://github.com/rails-api/active_model_serializers/pull/1252#issuecomment-159810477).
+
+We can take advantage of the scope to customize the objects returned based
+on the current user (scope).
+
+For example, we can limit the posts the current user sees to those they created:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
+
+  # scope comments to those created_by the current user
+  has_many :comments do
+    object.comments.where(created_by: current_user)
+  end
+end
+```
+
+##### Controller Authorization Context
+
+In the controller, the scope/scope_name options are equal to
+the [`serialization_scope`method](https://github.com/rails-api/active_model_serializers/blob/d02cd30fe55a3ea85e1d351b6e039620903c1871/lib/action_controller/serialization.rb#L13-L20),
+which is `:current_user`, by default.
```

Comment:
The controller defineds

```ruby
      class_attribute :_serialization_scope
      self._serialization_scope = :current_user

    def serialization_scope
      send(_serialization_scope) if _serialization_scope &&
        respond_to?(_serialization_scope, true)
    end
      def serialization_scope(scope)
        self._serialization_scope = scope
      end
    end

```

then serialization resource calls

```ruby
        serializable_resource.serialization_scope ||= serialization_scope
        serializable_resource.serialization_scope_name = _serialization_scope
```

notice that `serialization_scope` is actually the return value of sending the `_serialization_scope` method in the controller (if that method exists). 

Those two setters correspond to the serializar options `scope` and `scope_name` respectively, which may also be passed in as params to render